### ISSUE TITLE
Cant Wait for Lawsuit from Dolby

### ISSUE THREAD

**Issue by Yotsubal** [WinampDesktop/winamp#19]
[This software is company confidential information and the property of
 Dolby Sweden AB, and can not be reproduced or disclosed in any form
 without written authorization of Dolby Sweden AB.
](https://github.com/WinampDesktop/winamp/blob/1be04037cdba95396eef7728134043ba3ee6fbf4/Src/aacPlus/aacplusdectypes.h#L8)

```

 This program is protected under international copyright laws as an
 unpublished work. Do not copy.

               (C) Copyright Coding Technologies (2004 - 2007)
                 (C) Copyright Dolby Sweden AB (2008 - 2009)
                             All Rights Reserved

 This software is company confidential information and the property of
 Dolby Sweden AB, and can not be reproduced or disclosed in any form
 without written authorization of Dolby Sweden AB.

 Those intending to use this software module for other purposes are advised
 that this infringe existing or pending patents. Dolby Sweden AB has no
 liability for use of this software module or derivatives thereof in any
 implementation. Copyright is not released for any means. Dolby Sweden AB
 retains full right to use the code for its own purpose, assign or sell the
 code to a third party and to inhibit any user or third party from using the
 code. This copyright notice must be included in all copies or derivative
 works.
 ```

**Comment by ZaDarkSide** [comment]
It's fixed `Jef` deleted it also, don't worry everything is fine.

**Comment by jprjr** [comment]
See also https://github.com/WinampDesktop/winamp/issues/17#issuecomment-2371984811 for a brief write-up on header files and copyright.

**Comment by jfmauguit** [comment]
I agree with you @jprjr but I'll wait for the team to come back to me, I'll re-add this piece of code if it's OK

**Comment by Yotsubal** [comment]
> I agree with you @jprjr but I'll wait for the team to come back to me, I'll re-add this piece of code if it's OK

I think you should close the repo and check everything else first if it's OK to share the third-party code before publishing the entire source code online, so this one will not happen again
https://github.com/WinampDesktop/winamp/issues/11

I mean github has history for deleted codes, and everyone can check the history

please correct me if I am wrong

**Comment by ZaDarkSide** [comment]
> > I agree with you @jprjr but I'll wait for the team to come back to me, I'll re-add this piece of code if it's OK
> 
> I think you should close the repo and check everything else first if it's OK to share the third-party code before publishing the entire source code online, so this one will not happen again #11
> 
> I mean github has history for deleted codes, and everyone can check the history
> 
> please correct me if I am wrong

@Yotsubal it's fine, don't worry about it, let `Jef` work with his team he will get back to you on that. Cat is out of the bag anyway, no point to try to put it back in.

**Comment by ZaDarkSide** [comment]
I also believe this is a duplicate issue and can be closed.

**Comment by jprjr** [comment]
> > I agree with you @jprjr but I'll wait for the team to come back to me, I'll re-add this piece of code if it's OK
> 
> I think you should close the repo and check everything else first if it's OK to share the third-party code before publishing the entire source code online, so this one will not happen again #11
> 
> I mean github has history for deleted codes, and everyone can check the history
> 
> please correct me if I am wrong

Just a minor point - it is possible to rewrite git history and completely scrub things.

Unless GitHub gets involved though - that rewritten history won't apply to forks (I think/assume?).

And of course you can't account for anybody that's cloned the repo locally.

Once you push any commits to an internet-hosted repo, you kind of have to assume everybody has a copy now, even if you scrub it entirely from the git history.

**Comment by ZaDarkSide** [comment]
> Regardless if you uploaded confidential information, you're still going to get sued. Because, even IF you deleted it from the repository.. You broke your agreement with the company and let the information become public.
> 
> I.E, It's in public space now.
> 
> ### YIKES.

Only if they really really care, which they don't, also they were just some header files, and they got deleted anyway.

**Comment by ElCapor** [comment]
> > Regardless if you uploaded confidential information, you're still going to get sued. Because, even IF you deleted it from the repository.. You broke your agreement with the company and let the information become public.
> > I.E, It's in public space now.
> > ### YIKES.
> 
> Only if they really really care, which they don't, also they were just some header files, and they got deleted anyway.

I hope they sue you and your whole crew 🙏 
What if it was private US Army code ???? A copyright is a copyright and must be respected !

**Comment by ZaDarkSide** [comment]
> > > Regardless if you uploaded confidential information, you're still going to get sued. Because, even IF you deleted it from the repository.. You broke your agreement with the company and let the information become public.
> > > I.E, It's in public space now.
> > > ### YIKES.
> > 
> > 
> > Only if they really really care, which they don't, also they were just some header files, and they got deleted anyway.
> 
> I hope they sue you and your whole crew 🙏 What if it was private US Army code ???? A copyright is a copyright and must be respected !

My replies are copyrighted you didn't ask for permission to quote my replies! My copyright is a copyright and must pe respected!

**Comment by KatieBun** [comment]
![image](https://github.com/user-attachments/assets/a4d29f51-1804-4e41-8695-7bb6ac00f0c6)

Legal team at Dolby have been contacted

**Comment by jprjr** [comment]
Having the grounds to sue and actually doing it are two different things. Most copyright enforcement is voluntary - if I don't want to get sued, I generally avoid infringing on others copying. And similarly - if my copyright has been violated, usually I just need to *threaten* with a lawsuit (a cease and desist letter) and that's enough to get people to stop wherever it is they're doing. 

It's not like there's a copyright police running around and filing lawsuits. The big media companies do a good job of making it *look* that way, by going around trying to find people to sue. But the point is, it isn't something that just automatically happens. Somebody, somewhere, has to actually file the lawsuit.

To file a lawsuit costs time and money, involves lawyers, etc. So you gotta balance what's the cost of actually going through with suing somebody, versus what you hope to get out of it. 

I think in a situation like this where it was clearly accidental and the infringer makes a good faith effort to fix it, it's probably not worth the time to go through with a full-blown lawsuit.

**Comment by Aerocatia** [comment]
@KatieBun Why? What is the point?

**Comment by KatieBun** [comment]
Because in particular, Fuck Radionomy, Targetspot and The Llama Group.

Also Dolby are particularly keen on following up legal action

**Comment by UnDMCA-owner** [comment]
here is the AAC+ source:
[aacPlus.zip](https://github.com/user-attachments/files/17148597/aacPlus.zip)
